web/geosearch/tests/test_user_context.py

```diff
@@ -1,11 +1,15 @@
 import json
 
+import pytest
 from flask import current_app as app
 from flask import g
 
 from datapunt_geosearch import db
 
+only_run_on_azure = pytest.mark.skipif(app.config["CLOUD_ENV"] == "AZURE")
```

Comment:
This is more like `never_run_on_azure` :P

Reply:
haha good one